include/openthread/ble_secure.h

```diff
@@ -407,6 +407,17 @@ otError otBleSecureSendApplicationTlv(otInstance *aInstance, uint8_t *aBuf, uint
  */
 otError otBleSecureFlush(otInstance *aInstance);
 
+/**
+ * @brief Gets the Install Code Verify Status during the current session.
```

Comment:
nit: we typically don't use the `@brief` tag.
```suggestion
 * Gets the Install Code Verify Status during the current session.
```